apps/hosts/views.py

```diff
@@ -199,7 +199,7 @@ def remove_self_from_challenge_host_team(request, challenge_host_team_pk):
         response_data = {'error': 'ChallengeHostTeam does not exist'}
         return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
     try:
-        challenge_host = ChallengeHost.objects.filter(user=request.user.id, team_name__pk=challenge_host_team_pk)
+        challenge_host = ChallengeHostTeam.objects.filter(created_by=request.user.id, pk=challenge_host_team_pk)
```

Comment:
@deshraj this was some real desperate work from my side 😂  Coming to the problem, as of now, we show the teams `created_by` user in challenge-host-teams page. Since, the user still (after his removal from `ChallengeHost`) remains the `created_by` of `ChallengeHostTeam` this is happening. Well, will be reverting back this commit :)